src/AppBundle/Controller/BlogController.php

```diff
@@ -126,4 +127,25 @@ public function commentFormAction(Post $post)
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/rss.xml", name="blog_rss")
+     * @Method("GET")
+     */
+    public function rssAction()
+    {
+        $posts = $this->getDoctrine()->getRepository(Post::class)->findLatest();
+
+        $response = new Response();
+        // Like the Request, the Response object has also a public headers property.
+        // This is a ResponseHeaderBag that has some nice methods for getting and
+        // setting response headers. The header names are normalized so that using
+        // Content-Type is equivalent to content-type or even content_type.
+        $response->headers->set('Content-Type', 'application/xhtml+xml');
```

Comment:
I'm not sure about this. It's great to show how this works in Symfony ... but it may look "horrible" to newcomers because they might think they always need to do that (but it's optional, because `$this->render()` takes care of that).